test/functional/lua/vim_spec.lua

```diff
@@ -1066,4 +1066,48 @@ describe('lua stdlib', function()
     eq({5,15}, exec_lua[[ return vim.region(0,{1,5},{1,14},'v',true)[1] ]])
   end)
 
-  end)
+  do
+    local function test_tabsize(tabsize, softtabstop, implicit_buf)
+      local buf, arg
+      if implicit_buf then
```

Comment:
You can just always use `0` as the buffer list for this test. You don't need to have two tests for both explicit and implicit.

Reply:
I thought it was worth it to make sure passing an explicit buffer works as well but maybe not :) Will update.